composer.json

```diff
@@ -134,6 +134,12 @@
             },
             "drupal/redirect": {
                 "Redirection issue when interface language is different from content language": "https://www.drupal.org/files/issues/2020-06-01/redirect-interface_language_different_from_content_language_2991423-13.patch"
+            },
+            "drupal/socialblue": {
+                "YANG-6475 Make join methods for groups re-usable": "https://www.drupal.org/files/issues/2021-12-29/re-use-group-join-methods-3256328-3.patch"
```

Comment:
@tbsiqueira, I agree with @OH17L that we can track the drupal.org issue by patch file name if it is named correctly, see:
- https://www.drupal.org/docs/develop/git/using-git-to-contribute-to-drupal/patch-and-merge-request-guidelines#naming-conventions

So, I think we can use any patch title for cases like this if the patch file is named correctly.

Reply:
I would suggest then to not add the Jira issue number, since it's irrelevant information for the open source community, but of course is just a suggestion